Directory.Build.targets

```diff
@@ -3,7 +3,7 @@
 -->
 <Project>
   <PropertyGroup>
-    <NoWarn>$(NoWarn);IL2026</NoWarn>
+    <NoWarn>$(NoWarn);IL2104</NoWarn>
```

Comment:
Hmm, trying this out locally the warnings still happen. Let me see what is going on and fix it.

Reply:
The issue is https://github.com/dotnet/runtime/issues/94799. Looks like this approach won't work for this class of warnings. 😞

Reply:
Thanks for taking a look at this!